Swift/FullTextFunction.swift

```diff
@@ -28,17 +28,47 @@ public final class FullTextFunction {
     ///
     /// - Parameter indexName: The index name.
     /// - Returns: The full-text rank function.
+    @available(*, deprecated, message: "Use FullTextFunction.rank(withIndex:) instead.")
     public static func rank(_ indexName: String) -> ExpressionProtocol {
-        return QueryExpression(CBLQueryFullTextFunction.rank(indexName))
+        return QueryExpression(CBLQueryFullTextFunction.rank(withIndexName: indexName))
```

Comment:
It's better to call to the new function?

Reply:
The objective-c version should do the same as well.

Reply:
you mean get the Expression.fullTextIndex(name:), and pass it to the new API?